### ISSUE TITLE
The icon for the edit is not very informative and can be removed

### ISSUE THREAD

**Issue by omimo** [michaelnixon/mm-file-support#24]

**Comment by subirranjan1** [comment]
@omimo Can you please send screenshots which icons need to be removed?

**Comment by omimo** [comment]
It's for when the user is logged in and it's there in a couple of different views.

![screenshot from 2016-02-20 16-25-52](https://cloud.githubusercontent.com/assets/581375/13200088/c63d9bee-d7ee-11e5-9933-5d11717f79e9.png)

**Comment by subirranjan1** [comment]
@omimo  Please check the screenshot.  Just my thought I wanted to convey: If icon is removed, then people might overlook "edit" action by the side of the group name. Also as of now, it follows the design for the edit button in the website. Do you think we need to remove the icon or do you have any other alternative? I will do as asked in your reply. Thanks.

![edit icon](https://cloud.githubusercontent.com/assets/1945796/13212221/4f33401a-d8f4-11e5-8f3c-058fc4682af1.png)

**Comment by omimo** [comment]
I agree with Subir. This also seems inconsistence with other action buttons/links we have, which they all use an icon. Maybe we need to choose a better icon?
